_docs-2/getting-started/design.md

```diff
@@ -88,6 +88,13 @@ Multiple Monitors can be run to provide hot-standby support in the face of failu
 forwarding of logs from remote hosts to the Monitor, only one Monitor process should be active
 at one time. Leader election will be performed internally to choose the active Monitor.
 
+### Audit
+Accumulo has a robust audit service that logs most table actions, both successful and 
+failed attempts.  Audit logs are not enabled by default, but can be turned 
+on by editing the conf/log4j-service.properties file.  Audit logs are written to the same 
+location as the master, monitor and tserver logs, unless redirected in the conf/log4j-
+service.properties file.  Audit logs can be configured for the shell in conf/log4j.properties.
```

Comment:
This information doesn't accurately represent auditing. Auditing merely uses our normal slf4j logging facilities, but with a named logger (named `org.apache.accumulo.audit`). This logger can be configured by the user in whatever way they normally configure loggers, which would depend on the specific logging implementation they've configured in their deployment.

They are not necessarily enabled or disabled by default, and editing that file does not necessarily enable them, and it is not true that audit logs are written by default to the same location as the other loggers.

The `conf/log4j-service.properties` file is merely an example. Having our documentation specifically written under the assumption that it is the one way to configure logging, conflicts with the reason a lot of the work was done for logging and scripts in 2.0: that reason being to reduce assumptions in our code about the deployment environment, and thereby giving users greater control.

Documentation should reflect the fact that logging is fully configurable by the user. As far as auditing goes, we should indicate the name of the named slf4j logger we send audit logs to, and point to an the log4j configuration file as an example of what a user could do... but we should not place greater emphasis on the example configuration file we ship than it deserves. After all, it won't even have any effect if they don't have log4j on their class path for their slf4j implementation.

Reply:
Chris,

Thanks for the comments, I attempted to capture the main points you stated in my update, but please let me know if there is something missing or wrong.

Maybe we should also update the Configuration Files page at the Log Configuration Files section and add a comment that the log files are samples and that the loggers in those files can be added to any existing log4j logging framework instead of using those files.  Let me know what you think about that, or if you think that should be a separate ticket.